MAUI/Picker/date-picker.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: DatePicker with .NET MAUI Picker control | Syncfusion
+description: Learn here all about date picker with Syncfusion .NET MAUI Picker (SfPicker) control.
+platform: maui
+control: SfDatePicker
+documentation: ug
+---
+
+# Date Picker in .NET MAUI Picker (SfDatePicker)
+
+This section explains about the date picker.
+
+## DatePicker customization
+
+The `SfDatePicker` supports to customize [DateFormat], [MinimumDate], [MaximumDate], [DayInterval], [MonthInterval], [YearInterval], [SelectedDate] properties by the following code.
```

Comment:
The `SfDatePicker` supports customizing the [DateFormat], [MinimumDate], [MaximumDate], [DayInterval], [MonthInterval], [YearInterval], and [SelectedDate] properties using the following code.

Reply:
Content modified